src/modules/twap/state/emulatedTwapOrdersAtom.ts

```diff
@@ -0,0 +1,45 @@
+import { atom } from 'jotai'
+
+import { OrderStatus } from 'legacy/state/orders/actions'
+
+import { tokensByAddressAtom } from 'modules/tokensList/state/tokensListAtom'
+import { walletInfoAtom } from 'modules/wallet/api/state'
+
+import { OrderWithComposableCowInfo } from 'common/types'
+
+import { twapOrdersListAtom } from './twapOrdersListAtom'
+
+import { TwapOrderStatus } from '../types'
+import { emulateTwapAsOrder } from '../utils/emulateTwapAsOrder'
+
+const statusesMap: Record<TwapOrderStatus, OrderStatus> = {
+  [TwapOrderStatus.Cancelled]: OrderStatus.CANCELLED,
+  [TwapOrderStatus.Expired]: OrderStatus.EXPIRED,
+  [TwapOrderStatus.Pending]: OrderStatus.PENDING,
+  [TwapOrderStatus.Scheduled]: OrderStatus.SCHEDULED,
+  [TwapOrderStatus.WaitSigning]: OrderStatus.PRESIGNATURE_PENDING,
```

Comment:
Maybe we should update the `OrderStatus` to be called `PENDING_SIGNATURE` since it's more generic than `PRESIGNATURE_PENDING` in this case, TWAPS won't use `presignatures`

Reply:
also, more aligned witht he label we give https://github.com/cowprotocol/cowswap/blob/4984095de90a45d23e61df93de79d2e6085d1b65/src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx#L49

Reply:
Yeah, I've cut a corner here, just to avoid creating a new status.
To make the code more unambiguous - it makes sense.

Reply:
I agree. Just make sure the contract/backend mapping is still respected as this is what called on their side I believe